FlashCards.obitom67/FlashCards.obitom67/StudySession.cs

```diff
@@ -0,0 +1,93 @@
+﻿
+using System.Data.SqlClient;
+using Dapper;
+using Spectre.Console;
+
+
+namespace FlashCards.obitom67
+{
+    internal class StudySession
+    {
+
+        int StudyId { get; set; }
+        int StackId { get; set; }
+        int CorrectQ {  get; set; }
+        int TotalQ { get; set; }
+        string Date { get; set; }
+
+
```

Comment:
Please read point 1 of the [Code Conventions](https://thecsharpacademy.com/article/58/code-conventions).